src/main/java/net/minestom/server/network/NetworkBufferTypes.java

```diff
@@ -505,6 +506,24 @@ public int available() {
                 final int ordinal = buffer.read(VAR_INT);
                 return Entity.Pose.values()[ordinal];
             });
+    static final TypeImpl<DeathLocation> DEATH_LOCATION = new TypeImpl<>(DeathLocation.class,
+            (buffer, value) -> {
+                if (value == null) {
+                    buffer.write(BOOLEAN, false);
+                }
+                else {
```

Comment:
Same line as the bracket

Reply:
You'd prefer `if (value == null) buffer.write(BOOLEAN, false);` ?

Reply:
The `else {` is in a different line, definitely confusing

Reply:
Ah, you're one of *those* :P I'll fix